.swiftlint.yml

```diff
@@ -0,0 +1,82 @@
+disabled_rules:
+  - cyclomatic_complexity
+
+  # This is disabled because it is triggering on the case where a block
+  # parameter is declared on a newline. This conflicts with our style:
+  # "For functions with many argument, specify each arg on new line and )
+  #  on final line"
+  - opening_brace
+
+  - todo
+  - variable_name
+opt_in_rules:
+  - empty_count
+  - force_unwrapping
+included: # paths to include during linting.
+excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Carthage
+  - Pods
+  - Source/ExcludedFolder
+  - Source/ExcludedFile.swift
+
+closing_brace: error
+closure_spacing: error
+colon: error
+comma: error
+# conditional_returns_on_newline
+control_statement: error
+# cyclomatic_complexity:
+empty_count: error
+explicit_init: error
+file_length:
+  warning: 500
+  error: 1000
+force_cast: error
+force_try: error
+force_unwrapping: error
+function_body_length:
+  warning: 40
+  error: 100
+function_parameter_count:
+  warning: 6
+  error: 8
+leading_whitespace: error
+legacy_cggeometry_functions: error
+legacy_constant: error
+legacy_constructor: error
+line_length:
+  warning: 140
+  error: 200
+mark: error
+# missing_docs
```

Comment:
I did think it would be better to leave a default value in them....but im reading this the second time around and the syntax is easier than i imagined. when i first read 'todo' it was confusing but now I realize it must just be warning or error on todos in code.